docs/7.migration/3.auto-imports.md

```diff
@@ -1,16 +1,16 @@
 # Auto Imports
```

Comment:
Might be worth searching for all occurences of _auto import_ (case-insensitive search) and replacing them with hyphenated form.
```suggestion
# Auto-imports
```